linera-ethereum/src/test_utils/mod.rs

```diff
@@ -117,26 +117,26 @@ impl SimpleTokenContractFunction {
 
     // Only the balanceOf operation is of interest for this contract
     pub async fn balance_of(&self, to: &str) -> anyhow::Result<U256> {
-        // 1: getting the simple_token
+        // Getting the simple_token
         let contract_address = self.contract_address.parse::<Address>()?;
         let simple_token =
             SimpleTokenContract::new(contract_address, self.anvil_test.provider.clone());
-        // 2: gettting the balance transaction stuff
+        // Fetching the balance transaction stuff
```

Comment:
Is this fetching anything? I think it's just creating the calldata?

Reply:
Ok, changed.